endpoints.sql

```diff
@@ -0,0 +1,35 @@
+drop table if exists project cascade;
+drop table if exists endpoints cascade;
+
+create table project(
+    project_id serial primary key not null,
+    project_name varchar(255) not null
+);
+
+create table endpoints(
+   endpoint_id serial primary key not null,
+   project_id serial references project(project_id) not null,
+   endpoint_url varchar(255) not null,
+   request_method varchar(255) not null
+);
+
+------------------------------------------------------------------
+        --Dumping table for project
+------------------------------------------------------------------
+insert into project(project_name) values('Time Entry System');
```

Comment:
what is this dummy data for? Generally shouldn't be inserted in the sql creation script, maybe have a separate script for testing?